readthedocs/donate/migrations/0005_add-geo-filters.py

```diff
@@ -1,32 +0,0 @@
-# -*- coding: utf-8 -*-
-from __future__ import unicode_literals
-
-from __future__ import absolute_import
-from django.db import models, migrations
-import django_countries.fields
```

Comment:
Can we remove this dependency? We can probably move this to the ext repo if so.

Reply:
Yep.